admin_guide/install/prerequisites.adoc

```diff
@@ -230,29 +234,52 @@ registry], which involves adjusting the `--insecure-registry` option
 accordingly.
 ====
 
+[[configuring-docker-storage]]
 *Configuring Docker Storage*
 
 Docker's default loopback storage mechanism is not supported for production use
 and is only appropriate for proof of concept environments. For production
-environments, you must create a thin-pool logical volume and re-configure docker
+environments, you must create a thin-pool logical volume and re-configure Docker
 to use that volume.
 
 You can use the *docker-storage-setup* script to create a thin-pool device and
 configure Docker's storage driver after installing Docker but before you start
 using it. The script reads configuration options from the
-*_/etc/sysconfig/docker-storage-setup_* file.
+*_/etc/sysconfig/docker-storage-setup_* file. Allocate at least 15 GB of space
+for this purpose no matter which option you choose.
 
-. Configure *docker-storage-setup* for your environment. There are three options
-available based on your storage configuration:
+[NOTE]
+====
+See the https://access.redhat.com/articles/1492923["Managing Storage with Docker
+Formatted Containers on Red Hat Enterprise Linux and Red Hat Enterprise Linux
+Atomic Host"] Knowledgebase article for more details about
+*docker-storage-setup* and basic instructions on storage management in Red Hat
+Enterprise Linux 7.
+====
 
-.. Create a thin-pool volume from the remaining free space in the volume group
-where your root file system resides; this requires no configuration:
+. Configure *docker-storage-setup* for your environment. There are three methods
+available based on your storage configuration:
+[[docker-storage-a]]
+.. [[docker-storage-a]]Create a thin-pool volume from the remaining free space in the volume group
+where your root file system resides by running the *docker-storage-setup*
+command with no arguments; this requires no configuration.
++
+[IMPORTANT]
+====
+This method will fail if you do not have unpartitioned space available. Ensure
```

Comment:
I'd mention that it will appear to work but will fail silently when it runs out of space.